Core/Classrooms/Models/GetClassroomSessionResponseDto.cs

```diff
@@ -14,9 +14,9 @@ public class GetClassroomSessionResponseDto
     public int Id { get; set; }
     public string Title { get; set; } = string.Empty;
     public string Description { get; set; } = string.Empty;
-    public int AuthorId { get; set; }
+    public int Author { get; set; }
```

Comment:
Looks like this is actually the name and not the id of the author on the timed version

Reply:
Good catch!